src/__experimental__/components/label/__snapshots__/label.spec.js.snap

```diff
@@ -12,7 +12,6 @@ exports[`Label renders the label 1`] = `
 <label
   className="c0"
   data-element="label"
-  width={30}
```

Comment:
we need this to act as a min width

Reply:
I don't think it does?

![Screenshot 2019-04-01 at 14 42 18](https://user-images.githubusercontent.com/4471057/55332033-6cdba780-548c-11e9-8a92-9529b036bb17.png)

The label goes full-width if I change `width` to `min-width`?

Reply:
oh yeah, my mistake 👍